libs/pageserver_api/src/config.rs

```diff
@@ -491,6 +491,8 @@ pub struct TenantConfigToml {
     /// Tenant level performance sampling ratio override. Controls the ratio of get page requests
     /// that will get perf sampling for the tenant.
     pub sampling_ratio: Option<Ratio>,
+    /// Capacity of relsize PITR cache
```

Comment:
nit: What's `PITR`? In any case, let's explain the context behind this cache in a comment.